app/views/test_reports/show.js.erb

```diff
@@ -0,0 +1,15 @@
+$('.detail_table').html("<%= j(render "detail_table", data_for_test_reports: @data_for_test_reports, select_option: @select_option, job: @job , round: @round) %>");
+
+var element = document.getElementsByName("check_status");
+Object.keys(element).forEach((select) => {
```

Comment:
Change class name asynchronously.
And the element will change the background color by css.